message_tagging_service/tagging_service.py

```diff
@@ -324,10 +322,16 @@ def handle(rule_defs, event_msg):
 
     logger.debug('Modulemd file is downloaded and parsed.')
 
-    rule_matches = list(filter(truth, (
-        RuleDef(rule_def).match(modulemd)
-        for rule_def in rule_defs
-    )))
+    rule_matches = []
+    for i, rule_def in enumerate(rule_defs, 1):
+        rd = RuleDef(rule_def)
+        logger.info('[%s] Checking rule definition: %s', i, rd.id)
+        match = rd.match(modulemd)
```

Comment:
You could throw the log statement in `RuleDef.match` so you don't have to rewrite this bit of code.

Reply:
This my original solution. However, current version allows to print logs like this:

```
2019-01-11 10:01:41,354- message_tagging_service.tagging_service - INFO - [1] Checking rule definition: Building blocks                           
2019-01-11 10:01:41,357- message_tagging_service.tagging_service - INFO - [1] Rule definition: Not Matched.                                       
2019-01-11 10:01:41,357- message_tagging_service.tagging_service - INFO - [2] Checking rule definition: Modular Python for Fedora                 
2019-01-11 10:01:41,358- message_tagging_service.tagging_service - INFO - [2] Rule definition: Matched.                                           
2019-01-11 10:01:41,358- message_tagging_service.tagging_service - INFO - [3] Checking rule definition: Modular updates for Fedora                
2019-01-11 10:01:41,358- message_tagging_service.tagging_service - INFO - [3] Rule definition: Matched.                                           
2019-01-11 10:01:41,359- message_tagging_service.tagging_service - INFO - [4] Checking rule definition: EPEL updates                              
2019-01-11 10:01:41,359- message_tagging_service.tagging_service - INFO - [4] Rule definition: Not Matched.                                       
2019-01-11 10:01:41,359- message_tagging_service.tagging_service - INFO - [5] Checking rule definition: Scratch builds                            
2019-01-11 10:01:41,359- message_tagging_service.tagging_service - INFO - [5] Rule definition: Not Matched.                                       
2019-01-11 10:01:41,359- message_tagging_service.tagging_service - INFO - [6] Checking rule definition: Development builds                        
2019-01-11 10:01:41,360- message_tagging_service.tagging_service - INFO - [6] Rule definition: Not Matched.                                       
2019-01-11 10:01:41,360- message_tagging_service.tagging_service - INFO - [7] Checking rule definition: Fallback                                  
2019-01-11 10:01:41,360- message_tagging_service.tagging_service - INFO - [7] Rule definition: Matched.                                           
2019-01-11 10:01:41,360- message_tagging_service.tagging_service - INFO - Tag build mariadb-10.4-20190106153700.a5b0195c with tag(s) f30-modular-up
dates-candidate, f30-modular-updates-candidate, modular-fallback-tag
```

I found the `[N]` part would make it easier to read and follow the logs.